src/dateutils.ts

```diff
@@ -67,8 +76,8 @@ export function weekDayNames(firstDayOfWeek = 0) {
   return weekDaysNames;
 }
 
-export function page(date: XDate, firstDayOfWeek = 0, showSixWeeks = false) {
-  const days = month(date);
+export function page(date: XDate, firstDayOfWeek = 0, showSixWeeks = false, numberOfWeeks: number | undefined = undefined) {
```

Comment:
Why not use `numberOfWeeks?: number` instead of giving it `number | undefined` type and then set it to undefined?

Reply:
Will fix